serpent-moltres/isotope-distribution/compact-homo

```diff
@@ -0,0 +1,83 @@
+/*This input file defines a hexagonal elementary cell for the MHTGR fuel compact.*/
```

Comment:
Add that this is the homogenous model in the description at top of file

Reply:
done